src/borg/testsuite/archiver/info_cmd.py

```diff
@@ -47,6 +49,13 @@ def test_info_json_of_empty_archive(self):
         info_repo = json.loads(self.cmd(f"--repo={self.repository_location}", "info", "--json", "--last=1"))
         assert info_repo["archives"] == []
 
+    def test_info_on_repository_with_storage_quota(self):
+        self.create_regular_file("file1", size=1024 * 80)
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION, "--storage-quota=1G")
+        self.cmd(f"--repo={self.repository_location}", "create", "test", self.input_path)
+        info_archive = self.cmd(f"--repo={self.repository_location}", "info")
+        assert "Storage quota" in info_archive
+
```

Comment:
what used to be `borg info REPO` for borg < 2.0 is now called `borg rinfo REPO`.